winui/Polar-Chart/Axis-Labels.md

```diff
@@ -213,7 +95,7 @@ The appearance of the axis labels can be customized by using the [LabelTemplate]
 
 {% highlight xaml %}
 
-<chart:SfPolarChart>
+<chart:SfPolarChart x:Name="chart">
 
```

Comment:
Don't set name for chart, use parent element for resource usage.

Reply:
template changes are moved to the parent grid.